src/ostorlab/agent/message/proto/v3/report/vulnerability/vulnerability_pb2.py

```diff
@@ -23,25 +24,598 @@
+      type=None,
+      create_key=_descriptor._internal_create_key),
+    _descriptor.EnumValueDescriptor(
+      name='POTENTIALLY', index=3, number=3,
+      serialized_options=None,
+      type=None,
+      create_key=_descriptor._internal_create_key),
+    _descriptor.EnumValueDescriptor(
+      name='HARDENING', index=4, number=4,
+      serialized_options=None,
+      type=None,
+      create_key=_descriptor._internal_create_key),
+    _descriptor.EnumValueDescriptor(
+      name='SECURE', index=5, number=5,
+      serialized_options=None,
+      type=None,
+      create_key=_descriptor._internal_create_key),
+    _descriptor.EnumValueDescriptor(
+      name='IMPORTANT', index=6, number=6,
+      serialized_options=None,
+      type=None,
+      create_key=_descriptor._internal_create_key),
+    _descriptor.EnumValueDescriptor(
+      name='INFO', index=7, number=7,
+      serialized_options=None,
+      type=None,
+      create_key=_descriptor._internal_create_key),
+    _descriptor.EnumValueDescriptor(
+      name='CRITICAL', index=8, number=8,
+      serialized_options=None,
+      type=None,
+      create_key=_descriptor._internal_create_key),
+  ],
+  containing_type=None,
+  serialized_options=None,
+  serialized_start=2650,
+  serialized_end=2776,
+)
+_sym_db.RegisterEnumDescriptor(_RISKRATING)
+
+RiskRating = enum_type_wrapper.EnumTypeWrapper(_RISKRATING)
+_METADATATYPEENUM = _descriptor.EnumDescriptor(
+  name='MetadataTypeEnum',
+  full_name='ostorlab.agent.message.proto.v3.report.vulnerability.MetadataTypeEnum',
+  filename=None,
+  file=DESCRIPTOR,
+  create_key=_descriptor._internal_create_key,
+  values=[
+    _descriptor.EnumValueDescriptor(
+      name='FILE_PATH', index=0, number=0,
+      serialized_options=None,
+      type=None,
+      create_key=_descriptor._internal_create_key),
+    _descriptor.EnumValueDescriptor(
+      name='CODE_LOCATION', index=1, number=1,
+      serialized_options=None,
+      type=None,
+      create_key=_descriptor._internal_create_key),
+    _descriptor.EnumValueDescriptor(
+      name='URL', index=2, number=2,
+      serialized_options=None,
+      type=None,
+      create_key=_descriptor._internal_create_key),
+    _descriptor.EnumValueDescriptor(
+      name='PORT', index=3, number=3,
+      serialized_options=None,
+      type=None,
+      create_key=_descriptor._internal_create_key),
+    _descriptor.EnumValueDescriptor(
+      name='LOG', index=4, number=4,
+      serialized_options=None,
+      type=None,
+      create_key=_descriptor._internal_create_key),
+  ],
+  containing_type=None,
+  serialized_options=None,
+  serialized_start=2778,
+  serialized_end=2858,
+)
+_sym_db.RegisterEnumDescriptor(_METADATATYPEENUM)
+
+MetadataTypeEnum = enum_type_wrapper.EnumTypeWrapper(_METADATATYPEENUM)
+HIGH = 0
+MEDIUM = 1
+LOW = 2
+POTENTIALLY = 3
+HARDENING = 4
+SECURE = 5
+IMPORTANT = 6
+INFO = 7
+CRITICAL = 8
+FILE_PATH = 0
+CODE_LOCATION = 1
+URL = 2
+PORT = 3
+LOG = 4
```

Comment:
How/Where are these used, for indexing?

Reply:
They are typically used as enumeration values. for more in formations please check this documentation : https://protobuf.dev/programming-guides/proto3/